### ISSUE TITLE
Update Post Pricing Widget

### ISSUE THREAD

**Issue by ksmoore125** [laterpay/laterpay-wordpress-plugin#990]
Since we made the changes per https://github.com/laterpay/laterpay-wordpress-plugin/issues/977 we should make this apparent on the post page as well.

### Case 1: Global Default Price = Make article free unless price is set on post page
Display the following on the post page widget

### Case 2: Global Default Price = Posts cannot be purchased individually
Display the following on the post page widget

### Case 3: Global Default Price = Set individual article default price
No change from current behavior

**Comment by ksmoore125** [comment]
### Issue 1: (Optional) Hide post pricing if post is free
_If this will take more than one hour please notify Kimberly before any work is done_

If Global Default Price = Make article free unless price is set on post page then please hide the pricing section within the post so that it matches the mockup above
![image](https://user-images.githubusercontent.com/41650164/47037400-76728c80-d144-11e8-8857-7cc2e4b76794.png)

### Issue 2: (Minor Bug) Global Default Pricing Changes Not Reflected until Page Refresh
**Steps to Reproduce**
1. Open the Pricing Tab
2. Change the Global Default Price
3. Click Save

**Observed Result**
The Global Default Price, message above Category Default Price (when applicable), and message on Post price panel do not change

4. Refresh the Pricing page
Note that all changes are made as desired.

**Example**
Step 0: Starting Point
![image](https://user-images.githubusercontent.com/41650164/47037631-00225a00-d145-11e8-93aa-4b86fc3c4ba6.png)
![image](https://user-images.githubusercontent.com/41650164/47037643-07e1fe80-d145-11e8-90db-c9b24a1f1b8c.png)

Steps 1-3: Changed to "FREE unless price is set on post page or by category"
(Note that the validation across the top is correct but does not match what is being displayed below)
![image](https://user-images.githubusercontent.com/41650164/47037679-23e5a000-d145-11e8-9a06-8d5fd0413fdd.png)

**Comment by thrijith** [comment]
Issue 1: (Optional) Hide post pricing if post is free
>All articles will be free by default; Time Passes & Subscriptions will only be displayed if the article matches a Category Default Price or has an Individual Article Price set on the Post page.

Reference: https://github.com/laterpay/laterpay-wordpress-plugin/issues/992#issue-368770113

If Option One is selected then the user has to be provided with an option to set a price on the Post Page right? In that case, we should not remove the pricing section. LMK if wrong.

---

Issue 2: (Minor Bug) Global Default Pricing Changes Not Reflected until Page Refresh

The warning message appearing below `Category Default Prices` section should be in *red*, Can you try clearing the cache or opening in a private window?

Please check the below GIF and let me know if the behaviour is as expected.

![ezgif com-video-to-gif 1](https://user-images.githubusercontent.com/13589980/47061072-a2961800-d1ed-11e8-8a47-59710001c5ec.gif)

The post edit page is not updated when the option is changed, I am not sure if that can be done. I'll check that if it's needed, although after changing the option if you check the post on fronted it shows price according to the updated behaviour.

**Comment by ksmoore125** [comment]
### Issue 1: 
That is correct but only if they click `Individual Price`. Showing the price as free in text but then displaying 0.50 below it is a bit confusing. If it is simpler, displaying the price as 0.00 when the price is free would also be acceptable.

### Issue 2: 
**After opening in an incognito window everything is looking great!**

In regards to:

> The post edit page is not updated when the option is changed, I am not sure if that can be done. I'll check that if it's needed, although after changing the option if you check the post on fronted it shows price according to the updated behaviour.

It was not updating after refreshing the page either, I had to close the page and re-open it for the change to be reflected. That was my only concern. **But it is working properly now that I have cleared my cache. No changes needed!**

**Comment by thrijith** [comment]
>Issue 1:
That is correct but only if they click Individual Price. Showing the price as free in text but then displaying 0.50 below it is a bit confusing. If it is simpler, displaying the price as 0.00 when the price is free would also be acceptable.

Cool. Will only display the pricing section if the `Inidvidual Price` or `Category Default Price` is selected.

@ksmoore125 This shouldn't take more than an hour, but let me know if ☝️ is fine, then I can move forward with it.

**Comment by ksmoore125** [comment]
Please proceed, thanks!
